.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,16 @@
+---
+name: 'Bug report'
+about: 'Create a bug report to help us improve'
+labels: 'bug'
+---
+
+Type here...
+
+---
+
+Please follow these steps to make it more efficient to solve your issue:
+
```

Comment:
We should add something here about including complete code to reproduce the problem

Reply:
I do have it in the last check box below. Do you want me to put it on top instead?

Reply:
Oh, I see. That should be fine!